modules/upgrade/pages/server-y-yast.adoc

```diff
@@ -12,7 +12,7 @@ After installing, restart {yast} to ensure the tool is available within {yast}.
 
 [NOTE]
 ====
-Before upgrading from {productname} 4.0 to 4.1, ensure you do not have any unpublished packages.
+Before upgrading from {productname} 4.1 to 4.2 or 4.0 to 4.2, ensure you do not have any unpublished packages.
```

Comment:
We dropped "unpublisched patches" in 4.1. So this note apply only when directly updating from 4.0 to 4.2.
4.1 to 4.2 is not affected.